shared/fs/common/path-item-action/menu-container.js

```diff
@@ -34,10 +36,19 @@ const mapDispatchToProps = (dispatch, {path, routePath}: OwnProps) => ({
   _confirmSendToOtherApp: () =>
     dispatch(FsGen.createSetPathItemActionMenuView({view: 'confirm-send-to-other-app'})),
   _copyPath: () => dispatch(ConfigGen.createCopyToClipboard({text: Constants.escapePath(path)})),
-  _delete: () => {
+  _deleteFile: () => {
     dispatch(FsGen.createDeleteFile({path}))
     dispatch(Constants.makeActionForOpenPathInFilesTab(Types.getPathParent(path), routePath))
   },
+  _deleteFolder: () => {
+    dispatch(
+      RouteTreeGen.createNavigateTo({
+        parentPath: [fsTab],
+        path: [{props: {path}, selected: 'reallyDelete'}],
+      })
+    )
+    dispatch(Constants.makeActionForOpenPathInFilesTab(Types.getPathParent(path), routePath))
+  },
```

Comment:
IIRC, we want the confirmation for files as well, not just folders. So files/folders are actually the same.